Integrations/AlienVault_OTX_v2/AlienVault_OTX_v2.py

```diff
@@ -0,0 +1,875 @@
+            'Pulses': {
+                'Description': raw.get('description'),
+                'Created': raw.get('created'),
+                'Author': {
+                    'Username': raw.get('author').get('username')
+                },
+                'ID': raw.get('id'),
+                'Name': raw.get('name'),
+                'Tags': raw.get('tags'),
+                'TargetedCountries': raw.get('targeted_countries')
+            }
+        }
+    }
+
+    ec = remove_none(ec)
+
+    return raw, ec
+
+
+def alienvault_get_pulse_details_command():
+    raw, ec = alienvault_get_pulse_details()
+
+    human_readable = ''
+    if ec.get('AlienVaultOTX'):
+        if ec.get('AlienVaultOTX').get('Pulses'):
+            if ec.get('AlienVaultOTX').get('Pulses').get('Author'):
+                # Table 1
+                human_readable = tableToMarkdown(t=ec['AlienVaultOTX']['Pulses']['Author'],
+                                                 name='Pulse author')
+            if ec.get('AlienVaultOTX').get('Pulses').get('Tags'):
+                # Table 2
+                human_readable = tableToMarkdown(t={'Tags': ec['AlienVaultOTX']['Pulses']['Tags']},
+                                                 name='Tags')
+            key_not_need = ['Author', 'Tags']
+            ec_other = {k: ec['AlienVaultOTX']['Pulses'][k] for k in ec['AlienVaultOTX']['Pulses'] if k not in key_not_need}
+            # Table 3
+            human_readable += tableToMarkdown(t=ec_other,
+                                              name='Pulses General')
+
+    return_outputs(readable_output=human_readable,
+                   outputs=ec,
+                   raw_response=raw)
+
+
+# url command
+def url():
+    api_path = {
+        'source': 'indicators',
+        'command': 'url',
+        'value': demisto.args().get('url'),
+        'section': 'general'
+    }
+
+    # raw_data
+    raw = http_request(source=api_path['source'],
+                       command=api_path['command'],
+                       value=api_path['value'],
+                       section=api_path['section'])
+    ec = {
+        'URL': {
+            'Data': raw.get('indicator')
+        },
+        'AlienVaultOTX': {
+            'URL': {
+                'Hostname': raw.get('hostname'),
+                'Domain': raw.get('domain'),
+                'Alexa': raw.get('alexa'),
+                'Whois': raw.get('whois')
+            }
+        },
+        'DBotScore': {
+            'Indicator': raw.get('indicator'),
+            'Score': dbot_score(raw.get('pulse_info')),
+            'Type': 'URL',
+            'Vendor': 'AlienVault OTX'
+        }
+    }
+
+    ec = remove_none(ec)
+    return raw, ec
+
+
+def url_command():
+    raw, ec = url()
+
+    human_readable = ''
+    # Table 1
+    if ec.get('URL'):
+        human_readable += tableToMarkdown(t=ec.get('URL'),
+                                          name='URL')
+    # Table 2
+    if ec.get('AlienVaultOTX'):
```

Comment:
this can be placed under the previous condition so the second condition in the block is useless.
or did you mean:
```
    if ec.get('AlienVaultOTX', {}).get('URL')
```